### PR TITLE
UIDATIMP-29: Add UI for jobs logs pane.

### PR DESCRIPTION
**Pull request by VictorSoroka1**

### CODE REVIEW

File: src/components/JobLogs/withJobLogsCellsFormatter.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  intlShape,
+} from 'react-intl';
+
+import { compose } from '../../utils';
+
+const withJobLogsCellsFormatter = WrappedComponent => {
+  return class extends Component {
+    static propTypes = {
+      intl: intlShape.isRequired,
+      formatter: PropTypes.object,
```
Comment (maximdidenkoepam) [inline src/components/JobLogs/withJobLogsCellsFormatter.js]: Do we need default value for this property?

File: src/components/JobLogs/withJobLogsCellsFormatter.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+      this.cellFormatters = {
+        ...props.formatter,
+        runBy: this.formatUser,
+        completedDate: this.formatEndedRunningDate,
+      };
+    }
+
+    formatUser({
+      runBy: {
```
Comment (maximdidenkoepam) [inline src/components/JobLogs/withJobLogsCellsFormatter.js]: Please use class property approach. Also in this case I suggest to move destructuring from declaration to function body.

File: src/components/JobLogs/withJobLogsSort.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  sortNums,
+  sortDates,
+  sortStrings,
+} from '../../utils/sort';
+
+const withJobLogsSort = WrappedComponent => {
+  return class extends Component {
+    static propTypes = {
+      intl: intlShape.isRequired,
+      formatter: PropTypes.object,
```
Comment (maximdidenkoepam) [inline src/components/JobLogs/withJobLogsSort.js]: Please add missed props to default props

File: src/components/JobLogs/withJobLogsSort.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      if (isSameField) {
+        sortDir = direction === sortDir ? 'descending' : sortDir;
+      }
+
+      const sortState = {
+        sort: fieldName,
+        direction: sortDir,
+      };
+
+      this.setState(() => sortState);
```
Comment (maximdidenkoepam) [inline src/components/JobLogs/withJobLogsSort.js]: `this.setState(sortState)`?

File: src/routes/Home.js [hunk 5]
```diff
@@ -1,16 +1,39 @@
...
 
 import Jobs from '../components/Jobs';
+import JobLogs from '../components/JobLogs';
 import ImportJobs from '../components/ImportJobs';
 
-export default class Home extends React.Component {
+class Home extends Component {
   static propTypes = {
     stripes: stripesShape.isRequired,
+    resources: PropTypes.shape({
```
Comment (maximdidenkoepam) [inline src/routes/Home.js]: Please add default props.

File: src/components/JobLogs/withJobLogsSort.js [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+        direction = 'ascending',
+      } = qs.parse(this.props.location.search.slice(1));
+
+      this.setState({
+        sort,
+        direction,
+      });
+    }
+
+    prepareLogsData(logs = []) {
```
Comment (maximdidenkoepam) [inline src/components/JobLogs/withJobLogsSort.js]: I suggest use class props as much as possible. Please update this component.

File: src/components/JobLogs/withJobLogsSort.js (hunk 4 above)
Comment (VictorSoroka1) [reply src/components/JobLogs/withJobLogsSort.js]: I did this way intentionally. I prefer this way of setting state.

https://medium.com/@wisecobbler/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1

File: src/components/JobLogs/withJobLogsCellsFormatter.js (hunk 2 above)
Comment (VictorSoroka1) [reply src/components/JobLogs/withJobLogsCellsFormatter.js]: Do you mean to use static?

**Pull request merged**